tests/e2e/e2e_ibc_test.go

```diff
@@ -118,49 +111,56 @@ func (s *E2ETest) TestIBCTokenTransfer() {
 		umeeQuota := sdk.NewCoin(appparams.BondDenom,
-			sdk.NewDecFromInt(tokenQuota).Quo(atomPrice).Mul(powerReduction).RoundInt(),
+			sdk.NewDecFromInt(tokenQuota).Quo(umeePrice).Mul(powerReduction).RoundInt(),
 		)
 
-		// send $90 UMEE from umee to gaia (ibc_quota will check)
-		// Note: receiver is null so hermes will default send to key_name (from config) of target chain (gaia)
-		sendUmee := mulCoin(umeeQuota, "0.9")
-		s.SendIBC(s.Chain.ID, setup.GaiaChainID, "", sendUmee, false, fmt.Sprintf(
-			"sending %s (less than token quota) ", sendUmee.String()))
-		s.checkOutflows(umeeAPIEndpoint, appparams.BondDenom, true, sdk.NewDecFromInt(sendUmee.Amount), appparams.Name)
-		s.checkSupply(gaiaAPIEndpoint, umeeIBCHash, sendUmee.Amount)
+		// << TOKEN QUOTA EXCCEED >>
+		// send $110 UMEE from umee to gaia (token_quota is 100$)
+		exceedUmee := mulCoin(umeeQuota, "1.1")
+		s.SendIBC(s.Chain.ID, setup.GaiaChainID, "", exceedUmee, true, "")
+		// check the ibc (umee) quota after ibc txs - this one should have failed
+		// supply don't change
+		s.checkSupply(gaiaAPIEndpoint, umeeIBCHash, math.ZeroInt())
 
 		// send $110 ATOM from umee to gaia
 		exceedAtom := mulCoin(atomQuota, "1.1")
 		// supply will be not be decreased because sending amount is more than token quota so it will fail
 		s.SendIBC(s.Chain.ID, setup.GaiaChainID, "", exceedAtom, true, "uatom from umee to gaia")
 		s.checkSupply(umeeAPIEndpoint, uatomIBCHash, atomFromGaia.Amount)
 
-		// send $110 UMEE from umee to gaia (token_quota is 100$)
-		exceedUmee := mulCoin(umeeQuota, "1.1")
-		s.SendIBC(s.Chain.ID, setup.GaiaChainID, "", exceedUmee, true, "")
-		// check the ibc (umee) quota after ibc txs - this one should have failed
-		s.checkSupply(gaiaAPIEndpoint, umeeIBCHash, math.ZeroInt())
+		// << BELOW TOKEN QUOTA >>
+		// send $90 UMEE from umee to gaia (ibc_quota will check)
+		// Note: receiver is null so hermes will default send to key_name (from config) of target chain (gaia)
+		sendUmee := mulCoin(umeeQuota, "0.9")
+		s.SendIBC(s.Chain.ID, setup.GaiaChainID, "", sendUmee, false, fmt.Sprintf(
+			"sending %s (less than token quota) ", sendUmee.String()))
+		s.checkOutflows(umeeAPIEndpoint, appparams.BondDenom, true, sdk.NewDecFromInt(sendUmee.Amount), appparams.Name)
+		s.checkSupply(gaiaAPIEndpoint, umeeIBCHash, sendUmee.Amount)
 
+		// << BELOW TOKEN QUTOA 40$ but ATOM_QUOTA (40$)+ UMEE_QUOTA(90$) >= TOTAL QUOTA (120$) >>
 		// send $40 ATOM from umee to gaia
 		atom40 := mulCoin(atomQuota, "0.4")
 		s.SendIBC(s.Chain.ID, setup.GaiaChainID, "", atom40, true, "below token quota but not total quota")
 		// supply will be not be decreased because sending more than total quota from umee to gaia
 		s.checkSupply(umeeAPIEndpoint, uatomIBCHash, atomFromGaia.Amount)
 
+		// ✅ << BELOW TOKEN QUTOA 5$ but ATOM_QUOTA (5$)+ UMEE_QUOTA(90$) <= TOTAL QUOTA (120$) >>
 		// send $15 ATOM from umee to gaia
-		sendAtom := mulCoin(atomQuota, "0.15")
+		sendAtom := mulCoin(atomQuota, "0.05")
```

Comment:
why do we change the amount here?

Reply:
just making sure the total sending quota is always less than TotalQuota (120$)